apps/web/src/utils/achievements.ts

```diff
@@ -50,7 +50,7 @@ export const getClaimableIfoData = async (account: string, t: TranslateFunction)
   const ifosList = (await getIfoConfig(ChainId.BSC)) || []
   const ifoCampaigns = ifosList.filter((ifoItem) => ifoItem.campaignId !== undefined)
 
-  const bscClient = publicClient({ chainId: ChainId.BSC })
+  const bscClient = getViemClients({ chainId: ChainId.BSC })
```

Comment:
Why not this to match other changes?
```suggestion
    const client = viemClients[ChainId.BSC]
```